BibBuddy/app/src/main/java/de/bibbuddy/LibraryFragment.java

```diff
@@ -79,6 +87,11 @@ public boolean onOptionsItemSelected(MenuItem item) {
     switch (item.getItemId()) {
       case R.id.menu_backup_library:
         setStoragePermission();
+
+        //use the same parameter in order to be able to write
+        //in a specific file that already exists
+        createBibFile("Download", "exampleBibFilename");
```

Comment:
yes, but in case that I select 2 shelves with names "shelf1" and "shelf2" , which name of the shelf should I use? 
1. If I select "shelf1" as name for my .bib file, then the .bib file name will be **"shelf1.bib"**. 
2. If I, after that, select only the books from "shelf1" to be exported, then the current .bib file (with name **"shelf1.bib"**) will be overwritten with book data only from "shelf1".

Reply:
Corresponds to my general confusion here. Is the option here really the best one for an export? There is a menu option within a shelf to "Export", I feel like this is more logical? And on the upmost level of the library there would be only the option to export whole library, which then creates the file "library.bib"? Again, @SarahKurek and @claudia3 what do you think?